doc/source/rllib/algorithm-config.rst

```diff
@@ -0,0 +1,245 @@
+
+.. testcode::
+
+    config.environment()
+
+See this :ref:`RL environment guide <rllib-environments-doc>` for more details.
+
+Learning rate `lr`
+~~~~~~~~~~~~~~~~~~
+
+Set the learning rate for updating your models through the ``lr`` argument to the
+:py:meth:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig.training` method:
+
+.. testcode::
+
+    config.training(lr=0.0001)
+
+Train batch size
+~~~~~~~~~~~~~~~~
+
+Set the train batch size, per Learner actor,
+through the ``train_batch_size_per_learner`` argument to the :py:meth:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig.training`
+method:
+
+.. testcode::
+
+    config.training(train_batch_size_per_learner=256)
+
+Discount factor `gamma`
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Set the `RL discount factor <https://www.envisioning.io/vocab/discount-factor?utm_source=chatgpt.com>`__
+through the ``gamma`` argument to the :py:meth:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig.training`
+method:
+
+.. testcode::
+
+    config.training(gamma=0.995)
+
+Scaling with `num_env_runners` and `num_learners`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Set the number of :py:class:`~ray.rllib.env.env_runner.EnvRunner` actors used to collect training samples
+through the ``num_env_runners`` argument to the :py:meth:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig.env_runners`
+method:
+
+.. testcode::
+
+    config.env_runners(num_env_runners=4)
+
+    # Also use `num_envs_per_env_runner` to vectorize your environment on each EnvRunner actor.
+    # Note that this option is only available in single-agent setups.
+    #  The Ray Team is working on a solution for this restriction.
+    config.env_runners(num_envs_per_env_runner=10)
+
+Set the number of :py:class:`~ray.rllib.core.learner.learner.Learner` actors used to update your models
+through the ``num_learners`` argument to the :py:meth:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig.learners`
+method. This should correspond to the number of GPUs you have available for training.
+
+.. testcode::
+
+    config.learners(num_learners=2)
+
+Disable `explore` behavior
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Switch off/on exploratory behavior
+through the ``explore`` argument to the :py:meth:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig.env_runners`
+method. To compute actions, the :py:class:`~ray.rllib.env.env_runner.EnvRunner` calls `forward_exploration()` on the RLModule when ``explore=True``
+and `forward_inference()` when ``explore=False``. The default value is ``explore=True``.
+
+.. testcode::
+
+    # Disable exploration behavior.
+    # When False, the EnvRunner calls `forward_inference()` on the RLModule to compute
+    # actions instead of `forward_exploration()`.
+    config.env_runners(explore=False)
+
+Rollout length
+~~~~~~~~~~~~~~
+
+Set the number of timesteps each :py:class:`~ray.rllib.env.env_runner.EnvRunner` steps through each of its env copies
```

Comment:
We should mention here that the rollout length has a relation to the train batch size b/c this can raise errors.

Reply:
done